vpr/src/base/region.cpp

```diff
@@ -35,7 +35,7 @@ bool Region::locked() {
 }
 
 bool Region::empty() {
-    return region_bounds.empty();
+    return region_bounds.xmax() < region_bounds.xmin() || region_bounds.ymax() < region_bounds.ymin() || (region_bounds.xmin() == -1 && region_bounds.ymin() == -1 && region_bounds.xmax() == -1 && region_bounds.ymax() == -1);
```

Comment:
instead of having the default check, change the default values of the region_bounds so that they fail this test -> xmax and ymax can be -1, and xmin ymin can be a big number

Reply:
another option - could also have this empty as calculating area